packages/node/src/generators/setup-docker/files/Dockerfile__tmpl__

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/node:lts-alpine
+
+WORKDIR <%= app %>
+
+RUN addgroup --system node && \
+          adduser --system -G node node
+
+COPY <%= buildLocation %> .
+RUN chown -R node:node .
```

Comment:
I'm having an issue with `addgroup node`. I guess it's already there in the image by default.

Also missing a `CMD`. This one worked for me.

```FROM docker.io/node:lts-alpine

WORKDIR /app

RUN addgroup --system acme \
  && adduser --system -G acme acme

COPY main.js .
RUN chown -R acme:acme .

CMD node main.js
```

Reply:
We can use the app/project name rather than `acme`.